tracing-subscriber/src/fmt/time/chrono_crate.rs

```diff
@@ -0,0 +1,161 @@
+use crate::fmt::format::Writer;
+use crate::fmt::time::FormatTime;
+
+/// Formats [local time]s and [UTC time]s with `FormatTime` implementations
+/// that use the [`chrono` crate].
+///
+/// [local time]: [`chrono::offset::Local`]
+/// [UTC time]: [`chrono::offset::Utc`]
+/// [`chrono` crate]: [`chrono`]
+
+/// The RFC 3339 format is used by default but a custom format string
+/// can be used. See [`chrono::format::strftime`]for details on
+/// the supported syntax.
+///
+/// [`chrono::format::strftime`]: https://docs.rs/chrono/0.4.9/chrono/format/strftime/index.html
+#[cfg_attr(docsrs, doc(cfg(feature = "chrono")))]
```

Comment:
nit, not important: since this is a private type, we don't actually _need_ this attribute on it --- it won't show up in the `tracing-subscriber` API docs regardless.

Reply:
it was previously public but i made it private, but forgot to remove this cfg. will fix.